### PR TITLE
Add documentation for inequality constraints with tolerance

### PR DESCRIPTION
**Pull request by RenderMichael**
Add documentation for inequality constraints with tolerance

### CODE REVIEW

File: docs/articles/nunit/writing-tests/constraints/GreaterThanConstraint.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Is.Positive // Equivalent to Is.GreaterThan(0)
 ...Using(IComparer comparer)
 ...Using<T>(IComparer<T> comparer)
 ...Using<T>(Comparison<T> comparer)
+...Within(object tolerance)
```
Comment (SeanKilleen) [inline docs/articles/nunit/writing-tests/constraints/GreaterThanConstraint.md:24]: @OsirisTerje just curious before I merge this -- is this this technically `object` here? or should we provide a few different examples (e.g. TimeSpan, decimal, etc.)

File: docs/articles/nunit/writing-tests/constraints/GreaterThanOrEqualConstraint.md [hunk 2]
```diff
@@ -31,4 +32,6 @@ Assert.That(7, Is.AtLeast(3));
 Assert.That(7, Is.GreaterThanOrEqualTo(7));
 Assert.That(7, Is.AtLeast(7));
 Assert.That(myOwnObject, Is.GreaterThanOrEqualTo(theExpected).Using(myComparer));
+Assert.That(myDateTime, Is.GreaterThanOrEqualTo(expectedDateTime).Within(TimeSpan.FromSeconds(1)))
+
```
Comment (SeanKilleen) [inline docs/articles/nunit/writing-tests/constraints/GreaterThanOrEqualConstraint.md:36]:
```suggestion
```

**Comment by SeanKilleen** [comment]
@RenderMichael this is a great addition, thanks! 🎉 

I'm going to merge this after getting @OsirisTerje's take on my question.

**Review by SeanKilleen — APPROVED** [review]

**Comment by SeanKilleen** [comment]
PS Don't worry about the failing build here; it's purely related to an issue with my attempt to show previews of the changes via Netlify. The site builds correctly. 👍

File: docs/articles/nunit/writing-tests/constraints/GreaterThanConstraint.md (hunk 1 above)
Comment (RenderMichael) [reply docs/articles/nunit/writing-tests/constraints/GreaterThanConstraint.md:24]: So the type is `object`, and gets interpreted at runtime. I took this example from the equality constraint.

Comment (RenderMichael) [reply docs/articles/nunit/writing-tests/constraints/GreaterThanConstraint.md:24]: I can add examples of tolerance using numeric types, which was possible before the DateTime PR this documentation change is related to, but never documented here.

**Pull request merged**